src/Chirp.Razor/Pages/UserTimeline.cshtml.cs

```diff
@@ -1,10 +1,16 @@
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
-using Chirp.Infrastructure;
 using Chirp.Core;
 using Microsoft.AspNetCore.Authorization;
 namespace Chirp.Razor.Pages;
 
+/*
+<Summary>
+This is the users timeline model
+This is where the user can see their own timeline and those they follow.
+</Summary>
```

Comment:
And again again :) , maybe add that razor pages links this with the UserTimeline.cshtml file.